src/Workspace/WorkspaceItem.elm

```diff
@@ -32,7 +33,12 @@ import Workspace.Zoom exposing (Zoom(..))
 type WorkspaceItem
     = Loading Reference
     | Failure Reference Http.Error
-    | Success Reference Item Zoom
+    | Success
+        Reference
+        { item : Item
+        , zoom : Zoom
+        , docFoldToggles : DocFoldToggles
+        }
```

Comment:
We now have a sub record instead of discreet fields. Adding fields to this later will be easier and its nice that they are named.